contracts/protocol/test/utils/exchange_wrapper.ts

```diff
@@ -275,6 +278,19 @@ export class ExchangeWrapper {
         );
         return data;
     }
+    public abiDecodeFillOrder(data: string): AbiDecodedFillOrderData {
+        // Lookup fillOrder ABI in exchange abi
+        const fillOrderAbi = _.find(this._exchange.abi, (value: AbiDefinition) => {
```

Comment:
Maybe:
```suggestion
        const fillOrderAbi = _.find(this._exchange.abi, { name: 'fillOrder' }) as MethodAbi;
```

We don't usually use that form but I feel like here it's more readable. It's not type safe, but the current solution is not either.